hazelcast-sql/src/main/java/com/hazelcast/jet/sql/impl/opt/physical/SortPhysicalRel.java

```diff
@@ -88,4 +89,31 @@ protected RelDataType deriveRowType() {
         return getCollation().getFieldCollations()
                 .stream().map(FieldCollation::new).collect(Collectors.toList());
     }
+
+    public boolean requiresSort() {
```

Comment:
Same logic will be required in `SortPhysicalRule`, it would be nice to somehow share it.